app/src/main/java/org/simple/clinic/summary/prescribeddrugs/DrugSummaryEvent.kt

```diff
@@ -10,3 +11,5 @@ class PatientSummaryUpdateDrugsClicked : DrugSummaryEvent() {
 }
 
 data class PrescribedDrugsLoaded(val prescribedDrugs: List<PrescribedDrug>) : DrugSummaryEvent()
+
+data class CurrentFacilityLoaded(val facility: Facility) : DrugSummaryEvent()
```

Comment:
Adding this function right now to `Update` function means we have to handle other events as well. That's why I migrated both the controller streams in one commit. This flow goes like this, user clicks on the edit prescriptions/medicines button, we load current facility, once current facility is loaded we open the edit screen. So if we were to place this event in update right away, we need to handle `PrescribedDrugsLoaded` & `PatientSummaryUpdateDrugsClicked`. So instead of using `noChange` in `Update` and leaving the controller code I have decided to migrate it all in one go.